frontends/web/src/app.tsx

```diff
@@ -116,14 +113,33 @@ class App extends Component<Props, State> {
             }
         });
 
-        getAccounts()
-            .then(accounts => this.setState({ accounts }, this.maybeRoute))
-            .catch(console.error);
+        getDeviceList()
```

Comment:
A comment here explaining why exactly the two calls are chained like this will help in the future, I think.

Reply:
You're right, the requests can be done in parallel, no reason to wait, changed back and squashed 
https://github.com/digitalbitbox/bitbox-wallet-app/compare/d27e84efd912e3a1814cf96d168e64aa45ed3323..1711d06eac771a604f93ac9d2f42085f39fcb534

Also tested BB01 pairing, sending, restore form backup and quickly tested BB02 one more time, will merge now.